lib/model.js

```diff
@@ -1081,7 +1081,7 @@ Model.find = function find (conditions, projection, options, callback) {
  */
 
 Model.findById = function findById (id, projection, options, callback) {
-  return this.findOne({ _id: id }, projection, options, callback);
+  return this.findOne({ _id: id || null }, projection, options, callback);
```

Comment:
There's a small problem with this; nobody will be able to query for documents with `_id` 0, false, etc.